scripts/next-data/generatePreBuildFiles.mjs

```diff
@@ -29,6 +34,25 @@ export const generateBlogYearPages = cachedBlogData =>
     .then(data => [...new Set(data)])
     .then(data => data.forEach(createBlogYearFile));
 
+/**
+ * Render markdown to html via mdx
+ * @param {string} body
+ * @returns {Promise<string>}
+ * @see {https://github.com/mdx-js/mdx/discussions/1985}
+ */
+export const renderMarkdown = async body => {
+  const { default: mdx } = await evaluate(body, {
+    ...runtime,
+    format: "md",
+    development: false,
+    // format: 'md' — treat file as plain vanilla markdown
+    // Need to add the following remark plugins to support GFM and frontmatter
+    // https://github.com/remarkjs/remark-gfm
+    // https://github.com/remarkjs/remark-frontmatter
+    remarkPlugins: [remarkGfm, remarkFrontmatter],
```

Comment:
At the least, I need remarkFrontmatter. `format: "md",` makes it treat the markdown as plain markdown (no gfm).

Reply:
We don't need github flavoured markdown afaik.